### PR TITLE
Feat: Add Jira plugin to Roadie website

### PR DESCRIPTION
**Pull request by lewtakm**
Plugin adds Jira plugin to Roadie marketing website.

### CODE REVIEW

**Comment by dtuite** [comment]
@lewtakm this needs to be rebased before I can review it really. We changed the whole plugin page recently. Could you rebase it and force push please.

**Comment by lewtakm** [comment]
@dtuite done

File: content/plugins/descriptions/jira.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+# Instructions for someone who wants to use this plugin.
+# languages used here must be listed in the .babelrc
+gettingStarted:
+    # What will this step accomplish?
+  - intro: Install the plugin into Backstage.
+    language: bash
+    code: yarn add @roadiehq/backstage-plugin-jira
+  - intro: Add proxy config
+    language: yaml
```
Comment (dtuite) [inline content/plugins/descriptions/jira.yaml:42]: Got any idea why the YAML syntax highlighting doesn't work?

**Review by dtuite — APPROVED** [review]
Approved, with one minor question.

File: content/plugins/descriptions/jira.yaml (hunk 1 above)
Comment (lewtakm) [reply content/plugins/descriptions/jira.yaml:42]: @dtuite weird, from what I know syntax highlighter in Gastby use Prismjs that supports YAML files by default so it should work correctly..

**Pull request merged**